dream_layer_backend/preset_manager.py

```
@@ -0,0 +1,257 @@
+import json
+import os
+import hashlib
+from datetime import datetime
+from typing import Dict, List, Optional, Any
+from dataclasses import dataclass, asdict
+from pathlib import Path
```

Ruff (0.12.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)